RomeALincei/ANLcr11.xml

```diff
@@ -0,0 +1,370 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="ANLcr11" xml:lang="en" type="mss">
+    <teiHeader xml:base="https://betamasaheft.eu/">
+        <fileDesc>
+            <titleStmt>
+                <title>Excerpts of Gadla Sāmuʾel, Miracles of St George, Gadla Gabra Masqal,
```

Comment:
maybe expand to Sāmuʾel za-Dabra Hālleluyā, Gabra Masqal za-Gutamālā, and ʾƎnṭonyos za-Sarāye for the clarity?